test/runQuery.test.ts

```diff
@@ -1,9 +1,11 @@
 import fetch from "jest-fetch-mock";
-import { connect } from "../src/main";
+import { connect, resultsToObjects } from "../src/main";
```

Comment:
Unrelated to the rpcRelay, but I noticed we were not testing the `resultsToObjects` helper

Reply:
Nice, good catch